File: ruby_project/刷点击量.rb
# 在浏览器输入框输入待刷的网址，打开页面
# require 'watir-webdriver'
require 'watir'
require 'irb'

# 定义函数
def shua
	for i in 1..10
		@browser.goto "http://baoding.offcn.com/html/2020/03/47748.html"
		 puts "浏览器内执行的第 #{i} 次"
		sleep 3
	end
end


# 循环次数
for i in 1..10
	# 打开浏览器
	@browser=Watir::Browser.new :chrome
	 puts "第 #{i} 次打开浏览器"
	   shua()
	# 退出浏览器
	@browser.quit
end

File: python_project/刷点击量.py
# 导入webdriver包
from selenium import webdriver
# 导入time函数
import time

def shua( ):
	for i in  range(1,3):
		# 转到目标网址
		dr.get("http://baoding.offcn.com/html/2020/03/47748.html")

		# 等待时间
		time.sleep(1)

		# 输出
		print("第",i,"次打开网址")


# d打开浏览器次数
for i in  range(1,3):
	# 打开浏览器
	# dr = webdriver.Firefox()
	dr = webdriver.Chrome()
	# dr = webdriver.Ie()
	# 输出
	print("第",i,"次打开浏览器")

	# 执行函数
	shua()
	# 退出浏览器
	dr.quit()


